xdsl/dialects/onnx.py

```diff
@@ -153,11 +153,11 @@ class Gemm(IRDLOperation):
     tensor_b = operand_def(TensorType[T])
     tensor_c = operand_def(TensorType[T])
 
-    alpha = opt_attr_def(FloatAttr)
-    beta = opt_attr_def(FloatAttr)
+    alpha = opt_attr_def(FloatAttr, attr_name="alpha")
+    beta = opt_attr_def(FloatAttr, attr_name="beta")
```

Comment:
```suggestion
    alpha = opt_attr_def(FloatAttr)
    beta = opt_attr_def(FloatAttr)
```
when the names match this is not necessary, I'd recommend removing these